src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -63,10 +57,22 @@ export const NavBarSchedule: FC<NavBarScheduleProps> = ({ isVisible }) => {
   const userEventIds =
     userWithId?.myPersonalizedSchedule ?? emptyPersonalizedSchedule;
 
-  const { isLoading, relatedVenues, relatedVenueIds } = useRelatedVenues({
+  const {
+    isLoading,
+    relatedVenues,
+    relatedVenueIds,
+    sovereignVenue,
+  } = useRelatedVenues({
     currentVenueId: venueId,
   });
 
+  const isScheduleStartDate: boolean = sovereignVenue?.scheduleStartDate
+    ? true
+    : false;
+  const today = fromUnixTime(
+    sovereignVenue?.scheduleStartDate || getUnixTime(new Date())
+  );
```

Comment:
This code isn't perfect, but cleans this up a little bit.

We can use `isTruth` to simplify the first line.
We use `??` to default undefined things rather than `||`.

```suggestion
  const hasScheduleStartDate: boolean = isTruthy(sovereignVenue?.scheduleStartDate);
  
  const scheduleStartDate = sovereignVenue?.scheduleStartDate ? fromUnixTime(sovereignVenue?.scheduleStartDate) ?? Date.now()
  );
```

We can't use `Date.now()` (or `new Date`) in the main component body like this, as it will change on every render. This would need to be done within the `useMemo` below most likely.

So I would probably just extract the `scheduleStartDate` similar to this here (though probably it should be named better in the database to imply that it is in utc seconds):

```suggestion
  const scheduleStartDate = sovereignVenue ?? {}
```

Then would use this within the `useMemo` below (see code suggestions below)

Reply:
@margulies You didn't answer my question in https://github.com/sparkletown/sparkle/pull/1419#discussion_r634894934 ?

> Why is there a new `scheduleStartDate` config item here, rather than re-using `venue.start_utc_seconds` as the PR description suggests?

Reply:
The answer is below — I intended this to be a way to easily modify the start date directly in the firestore database. The interfact for putting in a Timestamp using the built-in calendar is much easier that searching utc_seconds. I've left it in as is, because with the type change in this revision, it seemed better not to modify `venue.start_utc_seconds`

Reply:
Replied in https://github.com/sparkletown/sparkle/pull/1419#discussion_r635721776

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1419/commits/b78373170c83bbc19324c8694b04ea35b9ab06ef